ee/ucp/kubernetes/layer-7-routing.md

```diff
@@ -19,32 +19,12 @@ Use an ingress controller when you want to:
 * Give your Kubernetes app an externally-reachable URL.
 * Load-balance traffic to your app.
 
-Kubernetes provides an NGINX ingress controller that you can use in Docker EE
-without modifications.
-Learn about [ingress in Kubernetes](https://v1-8.docs.kubernetes.io/docs/concepts/services-networking/ingress/).
+Kubernetes provides an NGINX ingress controller that can be used in Docker EE, but is not officially supported. Modifications are typically required based on your environment.
+Learn about [ingress in Kubernetes](https://v1-11.docs.kubernetes.io/docs/concepts/services-networking/ingress/). 
 
 ## Create a dedicated namespace
 
-1.  Navigate to the **Namespaces** page, and click **Create**.
-2.  In the **Object YAML** editor, append the following text.
-    ```yaml
-    metadata:
-      name: ingress-nginx
-    ```
-
-    The finished YAML should look like this.
-
-    ```yaml
-    apiVersion: v1
-    kind: Namespace
-    metadata:
-      name: ingress-nginx
-    ```
-3.  Click **Create**.
-4.  In the **ingress-nginx** namespace, click the **More options** icon,
-    and in the context menu, select **Set Context**.
-
-    ![](../images/deploy-ingress-controller-1.png){: .with-border}
+Because Kubernetes role based access control (RBAC) is supported, download the [Kubenetes YAML file](https://github.com/kubernetes/ingress-nginx/blob/master/deploy/mandatory.yaml) to create a dedicated namespace and default service account. 
 
 ## Create a grant
```

Comment:
`Create a grant` section can be dropped entirely.

Reply:
OK - thank you!